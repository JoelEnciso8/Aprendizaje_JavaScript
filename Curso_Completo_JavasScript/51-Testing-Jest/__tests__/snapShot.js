const cliente = {
    nombre: 'Joel Steve Enciso Bravo',
    balance: 900,
    tipo: 'Premium'
};

    describe('Testing Cliente',()=>{
        test('Joel Enciso',()=>{
            expect(cliente).toMatchSnapshot();
        });
        
    });
