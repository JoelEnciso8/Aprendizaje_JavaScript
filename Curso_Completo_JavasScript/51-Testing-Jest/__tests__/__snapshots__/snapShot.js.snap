// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Cliente Joel Enciso 1`] = `
{
  "balance": 900,
  "nombre": "Joel Steve Enciso Bravo",
  "tipo": "Premium",
}
`;
